rebel-readline/src/rebel_readline/clojure/line_reader.clj

```diff
@@ -790,15 +789,6 @@
                      "clojure-indent-line")
     (key-binding (KeyMap/ctrl \I) "clojure-indent-or-complete")))
 
-(defn bind-inserts [km-name]
-  (doto km-name
-    (key-binding (KeyMap/range " -~") "clojure-self-insert")
-    ;; the range behavior above overwrites all the bindings in the range
-    ;; so this keeps the oringinal bracket matching behavior
-    (key-binding ")" LineReader/INSERT_CLOSE_PAREN)
```

Comment:
as "clojure-self-insert" is no longer overwritting other bindings I think we can get rid of these too.